README.md

```diff
@@ -3,8 +3,8 @@
 
 # Morganey [![Status Ventis](https://img.shields.io/badge/status-ventis-yellow.svg)](https://github.com/ForNeVeR/andivionian-status-classifier)
 
-It's the [Lambda Calculus][wiki-lambda-calculus] Interpreter and it
-interprets lambda terms! \o/
+[Lambda Calculus][wiki-lambda-calculus] based programming language
+that recognises Church encoded structures.
```

Comment:
You mix British English (e.g. this `recognise` instance) and American English (e.g. `favorite` below). My vote for American dialect, so I'd replace this `recognise` with `recognize`.